app/assets/stylesheets/materialize.css

```diff
@@ -5118,27 +5118,27 @@ nav {
 
 @font-face {
   font-family: "Roboto";
-  src: url("../font/roboto/Roboto-Thin.woff2") format("woff2"), url("../font/roboto/Roboto-Thin.woff") format("woff"), url("../font/roboto/Roboto-Thin.ttf") format("truetype");
+  src: url("/assets/roboto/Roboto-Thin.woff2") format("woff2"), url("/assets/roboto/Roboto-Thin.woff") format("woff"), url("/assets/roboto/Roboto-Thin.ttf") format("truetype");
   font-weight: 200; }
 
 @font-face {
   font-family: "Roboto";
-  src: url("../font/roboto/Roboto-Light.woff2") format("woff2"), url("../font/roboto/Roboto-Light.woff") format("woff"), url("../font/roboto/Roboto-Light.ttf") format("truetype");
+  src: url("/assets/roboto/Roboto-Light.woff2") format("woff2"), url("/assets/roboto/Roboto-Light.woff") format("woff"), url("/assets/roboto/Roboto-Light.ttf") format("truetype");
   font-weight: 300; }
 
 @font-face {
   font-family: "Roboto";
-  src: url("../font/roboto/Roboto-Regular.woff2") format("woff2"), url("../font/roboto/Roboto-Regular.woff") format("woff"), url("../font/roboto/Roboto-Regular.ttf") format("truetype");
+  src: url("/assets/roboto/Roboto-Regular.woff2") format("woff2"), url("/assets/roboto/Roboto-Regular.woff") format("woff"), url("/assets/roboto/Roboto-Regular.ttf") format("truetype");
   font-weight: 400; }
 
 @font-face {
   font-family: "Roboto";
-  src: url("../font/roboto/Roboto-Medium.woff2") format("woff2"), url("../font/roboto/Roboto-Medium.woff") format("woff"), url("../font/roboto/Roboto-Medium.ttf") format("truetype");
+  src: url("/assets/roboto/Roboto-Medium.woff2") format("woff2"), url("/assets/roboto/Roboto-Medium.woff") format("woff"), url("/assets/roboto/Roboto-Medium.ttf") format("truetype");
   font-weight: 500; }
 
 @font-face {
   font-family: "Roboto";
-  src: url("../font/roboto/Roboto-Bold.woff2") format("woff2"), url("../font/roboto/Roboto-Bold.woff") format("woff"), url("../font/roboto/Roboto-Bold.ttf") format("truetype");
+  src: url("/assets/roboto/Roboto-Bold.woff2") format("woff2"), url("/assets/roboto/Roboto-Bold.woff") format("woff"), url("/assets/roboto/Roboto-Bold.ttf") format("truetype");
   font-weight: 700; }
 
 a {
```

Comment:
Notice the file path on all fonts is now dynamically generated when we compile our assets upon booting the server. This was the biggest problem. To include our `font` directory in the asset pipeline, I also had to write this line in `config/application.rb`:
```ruby
config.assets.paths << Rails.root.join("app", "assets", "font")
```